package-lock.json

```diff
@@ -24644,8 +24644,8 @@
                 "vite": "^5.4.2"
             },
             "peerDependencies": {
-                "@babylonjs/core": "^7.13.2",
-                "@babylonjs/loaders": "^7.0.0"
+                "@babylonjs/core": "^7.23.0",

```

Comment:
should it be the next version instead ? so 7.24.0 if merged tonight ?

Reply:
Yes, but I can't do that or it breaks the build. @RaananW says the only way is to put the wrong version, publish a build, fix the version, publish another build.

Reply:
oh yes cause it will install it before publishing at the end. so good luck with the double PR :-)